<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressKeyItem</class>
 <widget class="QWidget" name="AddressKeyItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>54</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddressKeyItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 238);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="m_lLabel">
     <property name="text">
      <string>Address / Key 1:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_leLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnAddressBook">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resource.qrc">
       <normaloff>:/png/address-book.png</normaloff>:/png/address-book.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnPaste">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resource.qrc">
       <normaloff>:/png/editpaste.png</normaloff>:/png/editpaste.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnRemove">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resource.qrc">
       <normaloff>:/png/remove.png</normaloff>:/png/remove.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
